backend/app/receive.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+import os.path
+import uuid
+
+from flask import request, abort
+import psycopg2
+from webargs import fields
+from webargs.flaskparser import use_args
+
+from . import app
+from .authorize_station import authorize_station
+from .utils import make_thumbnail
+
+cfg = app.config["database"]
+
+@app.route('/receive', methods=["POST",])
```

Comment:
An explanation is needed what's the expected syntax is.

Reply:
What I should explain here?
```methods``` parameter requires list of available HTTP methods. In Python when list has only one item then after this item should be trailing comma.

Reply:
My comment was about the whole /receive point. If I want to use this API, I should send a POST command. But how do I pass the parameters? In http headers? If so, what are the names?

Reply:
I saw the requests.http file. This is enough for explanation.